From be98fd5cb8d3cc2a4f504279bb903b0e76802a09 Mon Sep 17 00:00:00 2001
From: Tatsuo ONISHI <onishi@latmos.ipsl.fr>
Date: Wed, 23 Nov 2022 12:09:51 +0100
Subject: [PATCH 32/52] (Update) Error message output format updated

TYPE : Update

Description:
       Output format for "model_config_rec % dx(i)" in share/module_check_a_mundo.F is
       updated to avoid "********" in the output

List of modified files:
   * share/module_check_a_mundo.F
---
 share/module_check_a_mundo.F | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/share/module_check_a_mundo.F b/share/module_check_a_mundo.F
index 1acb3bda..8e95f571 100644
--- a/share/module_check_a_mundo.F
+++ b/share/module_check_a_mundo.F
@@ -273,7 +273,7 @@
                                 model_config_rec % max_dom, factor )
          model_config_rec % dx(i) = model_config_rec % dx(1) / REAL(factor)
          model_config_rec % dy(i) = model_config_rec % dy(1) / REAL(factor)
-         WRITE(wrf_err_message,FMT='(A,I2,A,F9.3,A)') 'Domain #',i,': dx = ',model_config_rec % dx(i),' m'
+         WRITE(wrf_err_message,FMT='(A,I2,A,F15.3,A)') 'Domain #',i,': dx = ',model_config_rec % dx(i),' m'
          CALL wrf_debug ( 0, wrf_err_message )
       END DO
 
-- 
2.31.1

